metashare/api/views.py

```diff
@@ -246,7 +246,13 @@ def commit(self, request, pk=None):
             )
         commit_message = serializer.validated_data["commit_message"]
         desired_changes = serializer.validated_data["changes"]
-        scratch_org.queue_commit_changes(request.user, desired_changes, commit_message)
+        target_directory = serializer.validated_data["target_directory"]
```

Comment:
Do we want to validate that `target_directory` is one of the options in `scratchorg.valid_target_directories`?